src/features/DebugSession.ts

```diff
@@ -51,6 +54,7 @@ export class DebugSessionFeature implements IFeature {
                 if (currentDocument.isUntitled) {
                     if (currentDocument.languageId === 'powershell') {
                         config.script = currentDocument.uri.toString();
+                        config.cwd = vscode.workspace.rootPath;
```

Comment:
Nope, that's why this logic will drop to the next case which checks whether the current file is untitled and sets it as `undefined` so that the cwd doesn't get set at all.

Though I just realize that this logic has a problem: if the user has an untitled file open but their launch config specifically references a different script file, the untitled file will cause the cwd of the configuration to be overwritten.  Sending a fix now.